api/controllers/TelevisionController.js

```diff
@@ -0,0 +1,328 @@
+/** 
+  * Gladys Project
+  * http://gladysproject.com
+  * Software under licence Creative Commons 3.0 France 
+  * http://creativecommons.org/licenses/by-nc-sa/3.0/fr/
+  * You may not use this software for commercial purposes.
+  * @author :: Pierre-Gilles Leymarie
+  */
+
+/**
+ * @apiDefine televisionParam
+ * @apiParam {Integer} [devicetype] DeviceType ID
+ * @apiParam {Integer} [room] Room ID 
+ * @apiDescription You can put the devicetype attribute OR the room attribute to determine in which room you want to control the television/which precise deviceType is playing.
+ */
+
+module.exports = {
+   
+
+   /**
+     * @api {post} /television/switchState switch state
+     * @apiName televisionSwitchState
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    switchState: function(req, res, next){
+        gladys.television.switchState(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {get} /television/state Get Current State
+     * @apiName televisionGetCurrentState
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * 
+     * @apiSuccess {Boolean} state State of the television
+     */
+    getState: function(req, res, next){
+        gladys.television.getState(req.query)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/channel Set channel
+     * @apiName televisionSetChannel
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * 
+     */
+    setChannel: function(req, res, next){
+        gladys.television.setChannel(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {get} /television/getChannel Get channel
```

Comment:
Same here, this is not RESTful!

Consider doing: 

`GET /television/channel`

Reply:
Fix this on commit c62f1694228a420b53efebcb015d877208a3c3d3